app/feature.go

```diff
@@ -0,0 +1,97 @@
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	ExportFeatureFlag = "enable-export"
+)
+
+func GetFeatureFlagConfigPath() string {
+	return ConfigDirFlag.Value + "/feature.json"
+}
+
+func GetFeatureFlagConfig(featureFlagConfigPath string) (map[string]bool, error) {
+	// create config file if not exist
+	if _, err := ioutil.ReadFile(featureFlagConfigPath); err != nil {
+		if err := ioutil.WriteFile(featureFlagConfigPath, []byte("{}"), 0644); err != nil {
+			return nil, err
+		}
+	}
+
+	content, err := ioutil.ReadFile(featureFlagConfigPath)
+	if err != nil {
+		return nil, err
+	}
+
+	var jsonData map[string]bool
```

Comment:
would prefer if this was a struct type so we can structure our data

Reply:
So the file will only contains feature flag and it's value:
{
"enable-export": true,
// in the future for other feature
"enable-XXX": false,
}

So this gives more flexibility for adding feature flag in the future. So I think Json would be fine?